### PR TITLE
feat: refactor BlueElectrum to Typescript

### PR DESCRIPTION
**Pull request by limpbrains**
had to change arguments order in multiGetTransactionByTxid to make Typescript works

only a couple of new `@ts-ignore`

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/oy2s2k6iyrn2mvkqots7owcrri?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/897)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/acaxh3qtxlbc7oextigh7l5pyi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/909)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uzjjkqsfksywlya4ub4jlc32f4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/910)

File: blue_modules/BlueElectrum.ts [hunk 1]
```diff
@@ -268,47 +366,24 @@ async function presentNetworkErrorAlert(usingPeer) {
...
 /**
  * Returns random electrum server out of list of servers
  * previous electrum server told us. Nearly half of them is
  * usually offline.
  * Not used for now.
- *
- * @returns {Promise<{tcp: number, host: string}>}
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-async function getRandomDynamicPeer() {
```
Comment (limpbrains) [inline blue_modules/BlueElectrum.ts:305]: unused, can be removed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3fz5mpzs7bqdq2myu6n4rahvlm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/924)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/wcjqsxewwb3gmsh4vpynzixs6q?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/940)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/szi6nxdmzqwatbdobasi3r2uym?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/968)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6owdt5xnbdr32xxbzjcsvfiblm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1022)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/aldozaummrju4wzyatdppkg57q?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1047)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by limpbrains** [comment]
@Overtorment Wake the fuck up samurai, we have PRs to merge

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k3ffh45nwxyk7xe7elojgvrh7m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1067)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2caue2vqavuapr5vfyzl2d5yxi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1111)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**